rules/do-you-display-information-consistently/rule.md

```diff
@@ -1,27 +1,25 @@
 ---
 type: rule
-archivedreason: 
 title: Do you display information consistently?
-guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 uri: do-you-display-information-consistently
-created: 2015-02-16T01:50:14.0000000Z
 authors: []
-related: []
+related:
+  - descriptive-links
 redirects: []
-
+created: 2015-02-16T01:50:14.000Z
+archivedreason: null
+guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 ---
-
-When redirecting users to another page make sure that the       text is consistent between the two pages.
+When redirecting users to another page, make sure that the text is consistent between the two pages.
 
 <!--endintro-->
 
+![Figure: Good Example - This text is taken from where the "More on SSW Chinafy" link goes](homepage-with-link-to-chinafy.png)
```

Comment:
@uly1 

- [ ] Do we need the "::: good   ...   :::" to get the green tick?

Reply:
yes i think so